hazelcast-jet-core/src/main/java/com/hazelcast/jet/pipeline/Sources.java

```diff
@@ -266,12 +273,13 @@ private Sources() {
      *
      * @param mapName the name of the map
      * @param predicateFn the predicate to filter the events. If you want to specify just the
-     *                    projection, use {@link Util#mapPutEvents} to pass only {@link
-     *                    com.hazelcast.core.EntryEventType#ADDED ADDED} and {@link
-     *                    com.hazelcast.core.EntryEventType#UPDATED UPDATED} events.
+     *                    projection, use {@link com.hazelcast.jet.Util#mapPutEvents} to pass
```

Comment:
no need for explicit imports here

Reply:
I've converted these to explicit imports so that I can use`com.hazelcast.jet.impl.util.Util` with import